M/MaximumIndependentSet/build_tarballs.jl

```diff
@@ -0,0 +1,41 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "MaximumIndependentSet"
+version = v"0.1.2"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/claud10cv/MaximumIndependentSet.git", "8a4d5f24f818800eb41ac4c7e8a3f663b96500d3")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/MaximumIndependentSet/src/
+Julia_PREFIX=${prefix}
+cmake -B build -DCMAKE_INSTALL_PREFIX=${prefix} -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release -DJulia_PREFIX=${Julia_PREFIX} -DJlCxx_DIR=$prefix/lib/cmake/JlCxx
+cmake --build build --parallel ${nproc}
+cmake --install build
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms(; 
+	exclude=[Platform("i686", "linux"; libc = "musl")
+		]
+)
```

Comment:
https://buildkite.com/julialang/yggdrasil/builds/9118#018e62ae-0e18-4c32-bb7b-99ef8051ce83/663-878
```
┌ Warning: /cache/build/yggy-amdci7-15/julialang/yggdrasil/M/MaximumIndependentSet/build/x86_64-linux-gnu/clphCAVm/x86_64-linux-gnu-libgfortran5-cxx11/destdir/lib/libmis.so contains std::string values!  This causes incompatibilities across the GCC 4/5 version boundary.  To remedy this, you must build a tarball for both GCC 4 and GCC 5.  To do this, immediately after your `platforms` definition in your `build_tarballs.jl` file, add the line:
│ 
│     platforms = expand_cxxstring_abis(platforms)
└ @ BinaryBuilder.Auditor /cache/julia-buildkite-plugin/depots/e2fd9734-29d8-45cd-b0eb-59f7104f3131/packages/BinaryBuilder/L8tBh/src/auditor/compiler_abi.jl:247
```
```suggestion
platforms = supported_platforms(; exclude=p->libc(p)=="musl" && arch(p)=="i686")
platforms = expand_cxxstring_abis(platforms)
```

Reply:
I see. I believe I implemented all the changes suggested (except for the last one that clashes with on of the other suggestions above)